src/osuTK.iOS/iOSGameView.cs

```diff
@@ -770,11 +770,11 @@ public void Run(double updatesPerSecond)
             Start ();
         }
 
-        public void RunWithPreferredFrameRateRange(CAFrameRateRange preferredFrameRateRange)
+        public void RunWithPreferredFrameRateRange((float min, float max, float pref) preferredFrameRateRange)
```

Comment:
Are we ever using custom params here? Might be easier to change `game_frame_rate_range` to be a get property or method and move/use in the innermost usage location if not?

Reply:
Yeah we don't, have merged `Run` and `RunWithPreferredFrameRateRange` together and inlined `game_frame_rate_range` to `CADisplayLinkTimeSource`.